src/main/scala/viper/gobra/translator/encodings/closures/ClosureSpecsEncoder.scala

```diff
@@ -48,7 +48,7 @@ protected class ClosureSpecsEncoder {
   def callToClosureGetter(func: in.FunctionMemberOrLitProxy, captured: Vector[(in.Expr, in.Parameter.In)] = Vector.empty)(ctx: Context): CodeWriter[vpr.Exp] = {
     val errorTransformers = register(in.ClosureSpec(func, Map.empty)(func.info))(ctx, func.info)
     for {
-      exp <- ctx.expression(in.PureFunctionCall(closureGetterFunctionProxy(func), captured.map(c => c._1), genericFuncType)(func.info))
+      exp <- ctx.expression(in.PureFunctionCall(closureGetterFunctionProxy(func), captured.map(c => c._1), genericFuncType, false)(func.info))
```

Comment:
it might be worth having a default value for the 'reveal' parameter

Reply:
I held off on that, because I wanted a compiler error for each instance such that I can take a look and think about places where it is used. Thinking about it though, having `reveal = false` should be safe.